### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by paulzay**
### What does this PR do?
Implements the second milestone
### Description of tasks to be completed?
1. Welcome players to the game
2. Get input from the players using gets.chomp
3. Display winner of the game

### CODE REVIEW

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag: 

Hi Team! :wave: 

Good attempt! :slightly_smiling_face: 

You are missing a couple of requirements which I will list below:

##### Required Changes [See this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/logic.png)
- the game asks the player to select from available moves
- the game informs player if the selected move is invalid (remember these features don't have to work per se but you need to address it in the flow)
- the game displays board after the player move
-  the game repeats all actions for the next player's move (this should be done with some sort of loop based on some terminating condition [recommended: while loop])


If you successfully address these, you'd gett the :+1: 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @paulzay and @luciano-ilha 
### You have done great and for making changes from the previous reviews :clap: However, you have a few issues to fix:-
- [ ] if the game is a stalemate, the message printed should be a draw, otherwise, player x wins
stalemate
- [ ] The board needs to change every time a player enters a value
board
- You also need a clear directive, which values should the players be entering i.e 1 - 9 or Xs and Os
direc

### Make the changes and submit for another code review. Good luck!

By Raymond
slack @iamraymond

File: bin/main [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+elsif board.include?(player_choice)
+  update_board.insert((player_choice - 1), 'X')
+
+  puts " #{update_board[0]} | #{update_board[1]} | #{update_board[2]} "
+  puts '-----------'
+  puts " #{update_board[3]} | #{update_board[4]} | #{update_board[5]} "
+  puts '-----------'
+  puts " #{update_board[6]} | #{update_board[7]} | #{update_board[8]} "
+
+end
```
Comment (majovanilla) [inline bin/main]: As required on the first review, the user's move should be inside a loop.
You can create something like this, and then adjust it in Milestone 3:
```game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team!!
## You did a *great* job on this project! 💜 

There's just one thing you left behind from the first review! Please check my comments on the code.
After you finish that, you're done!

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: bin/main [hunk 2]
```diff
@@ -20,45 +20,48 @@ puts " #{board[3]} | #{board[4]} | #{board[5]} "
...
-  puts 'Invalid position, choose again!'
-elsif board.include?(player_choice)
-  update_board.insert((player_choice - 1), 'X')
-
-  puts " #{update_board[0]} | #{update_board[1]} | #{update_board[2]} "
-  puts '-----------'
-  puts " #{update_board[3]} | #{update_board[4]} | #{update_board[5]} "
-  puts '-----------'
-  puts " #{update_board[6]} | #{update_board[7]} | #{update_board[8]} "
+while update_board.length < 9
```
Comment (suretrust) [inline bin/main:23]: You may need to change this condition in the next milestone.

**Review by suretrust — APPROVED** [review]
Hi team, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future milestones and happy coding!_ 🎉

**Pull request merged**